authority/provisioner/aws.go

```diff
@@ -271,7 +272,7 @@ func (p *AWS) Init(config Config) (err error) {
 func (p *AWS) AuthorizeSign(ctx context.Context, token string) ([]SignOption, error) {
 	payload, err := p.authorizeToken(token)
 	if err != nil {
-		return nil, err
+		return nil, errs.Wrap(http.StatusInternalServerError, err, "aws.AuthorizeSign")
```

Comment:
This should default to Unauthorized, here and all the `Authorize*` in the provisioners.
In fact, we can also do this in the caller of these methods, so we don't need to do it explicitly in the provisioners unless it's a real internal server error.

Reply:
I think it should default to `InternalServerError`. That way if it's incorrect we will at least be alerted and fix it. My thinking is that everything downstream should be throwing the correct error. If not, then this will catch it and alert us. Whereas a `401` wouldn't alert us because those happen pretty frequently for plenty of good reasons.